src/ProcessTable.cpp

```diff
@@ -74,9 +78,13 @@ static ProcessState stateToProcessState(char state) {
   }
 }
 
-ProcessInfo getProcessInfo(const std::string& pid) {
-  std::string commPath = "/proc/" + pid + "/comm";
-  std::string statusPath = "/proc/" + pid + "/status";
+ProcessInfo getProcessInfo(const std::string_view pid) {
+  std::string commPath = "/proc/";
+  commPath += pid;
+  commPath += "/comm";
```

Comment:
можно заменить на модный std::format)

```suggestion
  std::string commPath = std::format("/proc/{}/comm", pid);
```